lib/global_uid/migration_extension.rb

```diff
@@ -23,7 +23,7 @@ def create_table(name, options = {}, &blk)
     end
 
     def drop_table(name, options = {})
-      if !GlobalUid::Base.global_uid_options[:disabled] && options[:use_global_uid] == true
+      if !GlobalUid.configuration.disabled? && options[:use_global_uid] == true
```

Comment:
weird that the check for uid_enabled is done two different ways.

Reply:
These are two different options. `GlobalUid.configuration.disabled?` is checking if the gem is disabled globally. `options[:use_global_uid] == false` is checking if it's disabled for that particular table. 

I agree that it's inconsistent now that the configuration lives in a class and we're not using a hash for both checks